### PR TITLE
feat: Hide WPA3 feature based on Kura property

### PR DESCRIPTION
**Pull request by pierantoniomerlino**
This PR adds a way to disable the new WPA3 feature based on a specific Kura property.

**Related Issue:** This PR fixes/closes N/A

**Description of the solution adopted:** The #5586 added the (experimental) support for WPA3 Wifi Security. This PR adds a Kura property (`kura.wpa3.wifi.security.enable`) that hides/shows the WPA3 entries on the Wireless tab of the webUI. Moreover, the same property inhibits/allows the application of a network configuration containing WPA3 related properties to the framework.
The default behavior is to hide the feature.

### CODE REVIEW

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabWirelessUi.java [hunk 1]
```diff
@@ -152,6 +158,7 @@ interface TabWirelessUiUiBinder extends UiBinder<Widget, TabWirelessUi> {
     private final NetworkTabsUi netTabs;
     private final ListDataProvider<GwtWifiHotspotEntry> ssidDataProvider = new ListDataProvider<>();
     private final SingleSelectionModel<GwtWifiHotspotEntry> ssidSelectionModel = new SingleSelectionModel<>();
+    // private boolean isWPA3WifiSecuritySupported;
```
Comment (nicolatimeus) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabWirelessUi.java]: Please remove this commented code

File: kura/test/org.eclipse.kura.core.system.test/src/main/java/org/eclipse/kura/core/system/test/SystemServiceTest.java [hunk 2]
```diff
@@ -375,4 +375,10 @@ public void shouldGetDefaultLogManagerProperty() {
         assertFalse(systemService.getDefaultLogManager().isPresent());
     }
 
+    @TestTarget(targetPlatforms = { TestTarget.PLATFORM_ALL })
```
Comment (nicolatimeus) [inline kura/test/org.eclipse.kura.core.system.test/src/main/java/org/eclipse/kura/core/system/test/SystemServiceTest.java:378]: Please update copyright header

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtDeviceServiceImpl.java [hunk 3]
```diff
@@ -195,9 +195,9 @@ public ArrayList<GwtGroupedNVPair> findSystemProperties(GwtXSRFToken xsrfToken)
         // kura properties
         SystemService systemService = ServiceLocator.getInstance().getService(SystemService.class);
         Properties kuraProps = systemService.getProperties();
-        SortedSet kuraKeys = new TreeSet(kuraProps.keySet());
+        SortedSet kuraKeys = new TreeSet(kuraProps.stringPropertyNames());
```
Comment (nicolatimeus) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtDeviceServiceImpl.java:198]: Please update copyright header

File: kura/org.eclipse.kura.nm/OSGI-INF/networkConfigurationService.xml [hunk 4]
```diff
@@ -25,4 +25,5 @@
    <reference bind="setDnsServerService" cardinality="1..1" interface="org.eclipse.kura.internal.linux.net.dns.DnsServerService" name="DNSService" policy="static" />
    <reference name="CryptoService" interface="org.eclipse.kura.crypto.CryptoService" bind="setCryptoService" unbind="unsetCryptoService" cardinality="1..1" policy="static"/>
    <reference name="KeystoreService" interface="org.eclipse.kura.security.keystore.KeystoreService" bind="setKeystoreService" unbind="unsetKeystoreService" cardinality="0..n" policy="dynamic"/>
+   <reference bind="setSystemService" cardinality="1..1" interface="org.eclipse.kura.system.SystemService" name="SystemService" policy="static"/>
```
Comment (nicolatimeus) [inline kura/org.eclipse.kura.nm/OSGI-INF/networkConfigurationService.xml:28]: Please update copyright header

**Comment by nicolatimeus** [comment]
Should we also filter away/prohibit connection to the networks that use WPA3 from the scan results in web ui when in station mode?

**Comment by pierantoniomerlino** [comment]
@nicolatimeus I was convinced to have already add that filter... but it seems that I lost that modification in some rebase. I'll add it.

**Review by nicolatimeus — APPROVED** [review]
Tested as working, build is failing

**Pull request merged**